pkgs/development/libraries/pinocchio/default.nix

```diff
@@ -24,6 +24,12 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-h4NzfS27+jWyHbegxF+pgN6JzJdVAoM16J6G/9uNJc4=";
   };
 
+  # test failure, ref https://github.com/stack-of-tasks/pinocchio/issues/2277
+  prePatch = lib.optionals (stdenv.isLinux && stdenv.isAarch64) ''
+    substituteInPlace unittest/add_pinocchio_unit_test(force) \
```

Comment:
The filename doesn't seem right...

Reply:
Hum, I can't even copy-paste at this hour. Thanks !